Lib/_collections_abc.py

```diff
@@ -508,6 +508,10 @@ def __getitem__(self, item):
             new_args = (t_args, t_result)
         return _CallableGenericAlias(Callable, tuple(new_args))
 
+    # TODO: RUSTPYTHON patch for common call
```

Comment:
```suggestion
    # XXX: RUSTPYTHON patch for common call
```
This is a subtle commenting convention for local patches. What does common call here mean? Is there related test? Otherwise adding one under `extra_tests` will be great.